client/signup/utils.js

```diff
@@ -72,11 +60,11 @@ export function getStepUrl( flowName, stepName, stepSectionName, localeSlug ) {
 }
 
 export function getValidPath( parameters ) {
-	const locale = getLocale( parameters ),
-		flowName = getFlowName( parameters ),
-		currentFlowSteps = flows.getFlow( flowName ).steps,
-		stepName = getStepName( parameters ) || currentFlowSteps[ 0 ],
-		stepSectionName = getStepSectionName( parameters );
+	const locale = parameters.lang;
```

Comment:
Should this be `?.lang` just in case?

Reply:
`parameters?.lang` guards against `parameters` being undefined. But it's always a non-null object. If there are no route params, it remains empty.